scss/decanter.scss

```diff
@@ -1,20 +1,39 @@
-@charset "UTF-8";
+@charset 'UTF-8';
 
+///
+// Load Order
```

Comment:
Unfortunately, until the next version of sass comes out we will have to be very careful with the load order of things. There is no (clean/easy/ideal) way to prevent multiple includes and variable dependency management until this new version comes out. 

Bourbon, Neat, Bitters, Refils and USA Gov standards all follow the same convention as listed below.
eg: https://github.com/18F/web-design-standards/blob/develop/src/stylesheets/uswds.scss

In order to ensure that all variables have been set, all mixins are available, and everything is ready for the elements and components we need to have them loaded and ready before we try and load 5. and 6.

Separation is also key for being able to use Decanter without duplicate markup.
Levels 1, 3, & 4 MUST NOT generate any markup when compiled.